casper/src/main/scala/coop/rchain/casper/SafetyOracle.scala

```diff
@@ -53,108 +51,82 @@ sealed abstract class SafetyOracleInstances {
     new SafetyOracle[F] {
```

Comment:
`def turanOracle` is misleading now. 
If I understand correctly you want to fall back to the clique oracle implementation to ease debugging. 
Are you intending to go back to the Turan oracle in the near future? If so, why remove it?
Wouldn't it be beneficial to have both implementations available so we can freely switch between them?

Reply:
No we won't go back to the turan oracle as far as I can see. We used to have it because we thought we might have all validators with equal weights but otherwise it is an inferior oracle.